packages/stitch/src/stitchSchemas.ts

```diff
@@ -53,15 +55,19 @@ export function stitchSchemas({
   }
 
   let schemaLikeObjects: Array<GraphQLSchema | SubschemaConfig | DocumentNode | GraphQLNamedType> = [];
+  const processedSubschemas: Map<GraphQLSchema | SubschemaConfig, GraphQLSchema | SubschemaConfig> = new Map();
 
-  subschemas.forEach(subschemaOrSubschemaArray => {
-    if (Array.isArray(subschemaOrSubschemaArray)) {
-      schemaLikeObjects = schemaLikeObjects.concat(subschemaOrSubschemaArray);
+  subschemas.flat().forEach(subschema => {
```

Comment:
Should the members be checked manually whether they are arrays? Flat seems to be failing us

Reply:
We actually no longer need to allow arrays to be passed, but we released a version I think that does...